### PR TITLE
Added support options to the bottom right

### PR DESCRIPTION
**Pull request by ghost**
I don't know if this is actually wanted, but I made these changes on my instance, so why not share it. Not wanted? No problem! BTW: Maybe the Mastodon image should leave here, cause there is no space left cause of the increased height of the getting started column.

### CODE REVIEW

File: app/javascript/mastodon/features/getting_started/index.js [hunk 1]
```diff
@@ -57,7 +57,7 @@ class GettingStarted extends ImmutablePureComponent {
...
+            <p><FormattedMessage id='getting_started.support' defaultMessage='{faq} • {userguide} • {apps}' values={{ faq: <a href="https://github.com/tootsuite/documentation/blob/master/Using-Mastodon/FAQ.md" target="_blank"><FormattedMessage id='getting_started.faq' defaultMessage='FAQ' /></a>, userguide: <a href="https://github.com/tootsuite/documentation/blob/master/Using-Mastodon/User-guide.md" target="_blank"><FormattedMessage id='getting_started.userguide' defaultMessage='User Guide' /></a>, apps: <a href="https://github.com/tootsuite/documentation/blob/master/Using-Mastodon/Apps.md" target="_blank"><FormattedMessage id='getting_started.appsshort' defaultMessage='Apps' /></a> }} /><br /><FormattedMessage id='getting_started.open_source_notice' defaultMessage='Mastodon is open source software. You can contribute or report issues on GitHub at {github}.' values={{ github: <a href="https://github.com/tootsuite/mastodon" target="_blank">tootsuite/mastodon</a> }} /></p>
```
Comment (ykzts) [inline app/javascript/mastodon/features/getting_started/index.js]: Can you add a message to [en.json](https://github.com/tootsuite/mastodon/blob/4fcc0d5ac9e734deeadf305d3fc4b9c6b41b0a84/app/javascript/mastodon/locales/en.json) too?

**Review by ykzts — CHANGES_REQUESTED** [review]

File: app/javascript/mastodon/features/getting_started/index.js (hunk 1 above)
Comment (Gargron) [inline app/javascript/mastodon/features/getting_started/index.js]: Run `yarn run manage:translations for it to happen automatically`

Comment (jeroenpraat) [inline app/javascript/mastodon/features/getting_started/index.js]: OK. I will do it by hand. And also do 2 paragraphs, that looks cleaner. Still I advise to remove the background image there and to remove the scrollbar, but I will do a request for that later. So you and others can decide about that separately.

```bash
~/live$ docker-compose run --rm web yarn run manage:translations
yarn run v0.22.0
error Command "manage:translations" not found.
```

**Comment by jeroenpraat** [comment]
I think it is ready to be merged. Although if approved.

**Review by ykzts — APPROVED** [review]

File: app/javascript/mastodon/features/getting_started/index.js (hunk 1 above)
Comment (ykzts) [inline app/javascript/mastodon/features/getting_started/index.js]: @jeroenpraat Can I rebase master?

Comment (jeroenpraat) [inline app/javascript/mastodon/features/getting_started/index.js]: @ykzts Sorry, I don't understand what you mean?

Comment (ykzts) [inline app/javascript/mastodon/features/getting_started/index.js]: @jeroenpraat `yarn run manage:translations` can not be executed unless it is a new master.

Comment (jeroenpraat) [inline app/javascript/mastodon/features/getting_started/index.js]: @ykzts Ok, good to know. But it is not needed anymore, cause I added the strings manually.

**Pull request merged**